pkg/gather/clusterconfig/operators.go

```diff
@@ -327,6 +417,45 @@ func collectContainerLogs(ctx context.Context, coreClient corev1client.CoreV1Int
 	return nil
 }
 
+// isHealthyOperator checks if operator ins't degraded or unavailable
+func isHealthyOperator(operator *configv1.ClusterOperator) bool {
+	for _, condition := range operator.Status.Conditions {
+		switch {
```

Comment:
wouldn't a simple `if` be sufficient here ?

Reply:
All right, that is true. Old code didn't check it, but... yeah, if.